### PR TITLE
Re factor 

### PR DESCRIPTION
**Pull request by ETLDevs**
Splited the code into modules - main, store, handlers and winningStates - and refatored the code

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  ReturnOneStep,
+  showRecord,
+  saveGame,
+  loadGame,
+} from "./handlers.js";
+
+chooseBoard(game, gameTools);
+gameMoves(game, gameTools);
+restart(game, gameTools);
+ReturnOneStep(game, ALERTS);
```
Comment (shlomogcap) [inline src/main.js:15]: why is it Capital case? in general we use Capital case for class (i.e. es6 classes) or for Components (like react component which is type of classes)....

File: src/handlers.js [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+      game.xTurn = true;
+      game.xState = [];
+      game.oState = [];
+      gameTools.savedGame = { x: [], o: [], xTurn: true};
+    });
+  });
+};
+
+export const ReturnOneStep = (game, ALERTS) => {
```
Comment (shlomogcap) [inline src/handlers.js]: use camelCase

File: src/store.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+export const game = {
```
Comment (shlomogcap) [inline src/store.js:1]: why do you still have game object in game.js file... is this one replace it? if it is so you should remove the other one...

File: src/main.js (hunk 1 above)
Comment (ETLDevs) [reply src/main.js:15]: it's just a typo. I'll fix it

File: src/handlers.js (hunk 2 above)
Comment (ETLDevs) [reply src/handlers.js]: same..

File: src/store.js (hunk 3 above)
Comment (ETLDevs) [reply src/store.js:1]: dont need it anymore. I thought to delete it after merging ...

**Review by shlomogcap — APPROVED** [review]

**Pull request merged**